nvflare/private/fed/app/client/sub_worker_process.py

```diff
@@ -310,11 +310,11 @@ def stop(self):
 def main(args):
     workspace = Workspace(args.workspace, args.client_name)
     app_custom_folder = workspace.get_client_custom_dir()
-    if os.path.isdir(app_custom_folder):
+    if os.path.isdir(app_custom_folder) and app_custom_folder not in sys.path:
         sys.path.append(app_custom_folder)
     configure_logging(workspace)
 
-    fobs_initialize()
+    fobs_initialize(workspace=workspace, job_id=args.job_id)
```

Comment:
Don't we also need to register ext modules?

Reply:
Added for multi-gpu sub_worker_process.py